spec/mailers/user_mailer_spec.rb

```diff
@@ -131,4 +131,29 @@
       end
     end
   end
+
+  describe "#claim_created_support_notification" do
+    subject(:claim_confirmation_email) { described_class.with(service: support_user.service).claim_created_support_notification(claim) }
+
+    context "when a claim has been submitted" do
+      let(:region) { create(:region, name: "A region", claims_funding_available_per_hour: Money.from_amount(53.60, "GBP")) }
+      let(:support_user) { create(:claims_support_user, :colin) }
+      let(:school) { create(:claims_school, region:) }
+      let(:claim) { create(:claim, reference: "123", school:) }
+
+      let!(:user_of_a_school) { create(:claims_user, email: "babagoli@gmail.com", user_memberships: [create(:user_membership, organisation: school)]) }
```

Comment:
```suggestion
      let!(:user_of_a_school) { create(:claims_user, email: "babagoli@gmail.com", user_memberships: [build(:user_membership, organisation: school)]) }
```

Could we use `build` here?